### PR TITLE
add regionOverride field to AWS integration, for overriding multiple regions

### PR DESCRIPTION
**Pull request by davidbloss**
Resolves [#505](https://github.com/OpsLevel/team-platform/issues/505)

### Checklist

- [ ] I have run this code, and it appears to resolve the stated issue.
- [ ] This PR does not reduce total test coverage
- [ ] This PR has no user interface changes or has already received approval from product management to change the interface.
- [ ] Does this change require a Terraform schema change?
  - If so what is the ticket or PR #
- [ ] Make a [changie](https://github.com/OpsLevel/opslevel-go/blob/main/CONTRIBUTING.md#changie-change-log-generation) entry that explains the customer facing outcome of this change

### CODE REVIEW

File: integration.go [hunk 1]
```diff
@@ -53,11 +54,12 @@ type IntegrationConnection struct {
 }
 
 type AWSIntegrationInput struct {
```
Comment (rocktavious) [inline integration.go:56]: It seems this struct is duplicative of whats in `input.go` ?  why?

**Review by rocktavious — APPROVED** [review]
LGTM - just not sure why we have duplicative "input" structs

File: integration.go (hunk 1 above)
Comment (davidbloss) [reply integration.go:56]: Yeah I noticed that... not sure what happened here but didn't want to yank out the duplicate before figuring out which is the "good" one

Comment (rocktavious) [reply integration.go:56]: Lets make sure we follow this up

**Pull request merged**